app/models/metric/long_term_averages.rb

```diff
@@ -62,7 +62,8 @@ def self.get_averages_over_time_period(obj, options = {})
       Metric::Aggregation::Process.average(c, nil, results[:avg], counts)
 
       begin
-        results[:dev][c]  = vals[c].stddev
+        results[:dev][c] = vals[c].length == 1 ? 0 : vals[c].stddev
```

Comment:
@isimluk Yes, more_core_extensions is intended to be a set of features that could potentially be moved to ActiveSupport.  However, based on http://ruby-doc.org/core-2.3.1/ZeroDivisionError.html and its limited documentation, I would actually expect it to raise `ZeroDivisionError` since it winds up dividing by 0, not 0.0.